examples/cross-contract-high-level/src/lib.rs

```diff
@@ -1,23 +1,24 @@
 use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
 use near_sdk::{
-    env,
-    ext_contract,
-    json_types::U128,
-    log,
-    near_bindgen,
-    AccountId,
-    Promise,
-    PromiseOrValue,
+    env, ext_contract, json_types::U128, log, near_bindgen, AccountId, Gas, Promise, PromiseOrValue,
 };
 
+const TGAS: u64 = 1_000_000_000_000;
```

Comment:
optionally this can be `Gas` type, and it has a `Mul<u64>` impl so usage below won't break

Reply:
I started with a `Gas` type, but the arithmetic got somewhat complicated. I ended up with calculations like:

```rs
Gas(2 * TGAS.0)
```

Is there a better way to do that?

Reply:
https://github.com/near/near-sdk-rs/commit/6bba3327c2e0439a3a2444819c35074c98dbe126 (it's a little finicky because of how the multiplication trait is setup in rust so the ordering matters)

Reply:
Thanks!